packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,157 @@ export const pageQuery = graphql`
 
 ### About Cards
 
-Cards are a specifically-styled container that group relatable content and actions.
-Card is an extremely flexible container that does not require specific components inside of it,
-allowing an implementer to compose as needed, but elements such as Title, Body,
-and Button or Anchor are commonly used. See Examples for common Card patterns.
-A Card does not currently handle any interactive events such as hover, click or focus,
-though children composed inside of it may commonly have event handlers.
+Cards are specifically-styled containers that group relatable content and actions. Card is an extremely 
+flexible container that does not require specific components inside of it. You can compose a Card to support your use case, 
+but elements such as [Heading](/components/heading), [Paragraph](/components/paragraph), and [Button](/components/button) or [Anchor](/components/anchor) are commonly used.
+See [When to use a Card](#when-to-use-a-card) and [Examples](#examples) for common Card patterns and dos/donts. A Card does not 
+currently handle any interactive events such as hover, click, or focus, though children 
+composed inside of it may commonly have event handlers.
```

Comment:
The last sentence could be broken out into its own paragraph. And take out the word "currently" since we don't have plans yet to add event handlers.